clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionState.java

```diff
@@ -0,0 +1,145 @@
+/**
```

Comment:
We may as well start using the new license header. I had to fix all of these when I submitted the PR for the message format changes.